models/habits.js

```diff
@@ -40,6 +40,11 @@ const HabitsModel = (sequelize, DataTypes) => {
       foreignKey: 'habitId',
       onDelete: 'CASCADE'
     });
+
+    Habits.hasMany(models.Notifications, {
```

Comment:
I think the notification should be more about the user than the habit, since it's a user that will get notified when certain actions occur on his habit(s).